dev_guide/deployments.adoc

```diff
@@ -364,17 +374,30 @@ the trigger is disabled.
 With the above example, when the `latest` tag value of the *origin-ruby-sample*
 image stream changes and the new image value differs from the current image
 specified in the deployment configuration's *helloworld* container, a new
-deployment is created using the new image for the *helloworld* container.
+replication controller is created using the new image for the *helloworld* container.
 
 [NOTE]
 ====
 If an `*ImageChange*` trigger is defined on a deployment configuration (with a
-`*ConfigChange*` trigger or with `automatic=true`) and the `*ImageStreamTag*`
-pointed by the `*ImageChange*` trigger does not exist yet, then the first
-deployment automatically starts as soon as an image is imported or pushed by a
-build to the `*ImageStreamTag*`.
+`*ConfigChange*` trigger and `automatic=false`, or with `automatic=true`) and the
+`*ImageStreamTag*` pointed by the `*ImageChange*` trigger does not exist yet, then
+the initial deployment process will automatically start as soon as an image is
+imported or pushed by a build to the `*ImageStreamTag*`.
 ====
 
+[[deployment-triggers-using-the-command-line]]
```

Comment:
reminder for future me: describe the `oc set image` here (it will trigger deployment and maybe used to manually controller what image will be deployed)